Api/Parameter/EnumParameter.php

```diff
@@ -18,11 +18,6 @@
  */
 class EnumParameter extends Parameter
 {
-    /**
-     * @var array
-     */
-    private $enum;
-
     /**
      * EnumParameter constructor.
      *
```

Comment:
Missing a @throws

Reply:
Instead of array, you can use mixed[] (it's better if phpdoc defines clearly the content of the array. array is not specific)